src/cli_common.c

```diff
@@ -318,8 +318,11 @@ void parseRedisUri(const char *uri, const char* tool_name, cliConnInfo *connInfo
     UNUSED(tls_flag);
 #endif
 
-    const char *scheme = "redis://";
-    const char *tlsscheme = "rediss://";
+    const char *scheme = "valkey://";
+    const char *tlsscheme = "valkeys://";
```

Comment:
I have a concerns about this. With Redis, using `rediss` was clearer to me that it was SSL because there is obviously a second s. With Valkey, I'm concerned `valkeys` will be confusing since it's not clear. I want us to least thing about it a bit more.

Reply:
@madolson think about it for a while then. :)

Adding "s" is the most common style for URI schemes, so I think `valkeys` is the right choice. Here are some from IANA's list:

```
aaa 		Diameter Protocol
aaas 		Diameter Protocol with Secure Transport
http 		Hypertext Transfer Protocol
https 		Hypertext Transfer Protocol Secure
msrp 		Message Session Relay Protocol
msrps 		Message Session Relay Protocol Secure
rtsp 		Real-Time Streaming Protocol (RTSP)
rtsps 		Real-Time Streaming Protocol (RTSP) over TLS
sip 		session initiation protocol
sips 		secure session initiation protocol
```

Reply:
Ok, I thought about it and will retract my concern :)

Reply:
I guess now would be the time to rethink the URI scheme if we wanted to do that though.

Reply:
> We can postpone it indefinitely and only support the redis scheme. Do you prefer that?

I'm leaning towards yes.

Reply:
We can consider `resp` as the scheme. RESP is the protocol after all. :)

Reply:
I like `resp` more.  In fact, I think "redis" was a minomer to begin with. It should've been a protocol name.

Reply:
I think the redis scheme contains more than what's specified in RESP.

An example URL `redis://user:password@host:port/dbnum` contains information for the AUTH command and the SELECT command, to be called after connecting. It doesn't contain the RESP version though (for HELLO). It's a bit ad-hoc all of it.

Reply:
It could have included various more things which are useful to do when connecting, like CLIENT SETNAME, CLIENT SETINFO LIBNAME, CLIENT SETINFO LIBVER, CLIENT NO-EVICT.

For example we could support them in a query string, such as

    ?name=banana&libname=valkey-py&libver=3.14.159&no-evict=1

The RESP version (HELLO N) very much affects what comes out from each command, so it's basically a different protocol. We could embed that in the scheme as `resp2:` vs `resp3:`.

.... or we can just keep the `redis` schemes just for backward compatibility. I'm not sure we need a scheme anyway, do we?

Reply:
As long as valkey-cli can accept "redis/rediss", I am fine with other options, "valkey" or "resp2/3". 

@zuiderkwast, you have a good point that the scheme implies more than just the wire protocol but also the specific client behavior so this is more of a call on the client side.